src/locale/locales/de/messages.po

```diff
@@ -249,7 +227,7 @@ msgstr "Alt-Text beschreibt Bilder für blinde und sehbehinderte Nutzer und hilf
 
 #: src/view/com/modals/VerifyEmail.tsx:124
 msgid "An email has been sent to {0}. It includes a confirmation code which you can enter below."
-msgstr "Eine E-Mail wurde an {0} gesendet . Sie enthält einen Bestätigungscode, den du unten eingeben kannst."
+msgstr "Eine E-Mail wurde an {0} gesendet. Sie enthält einen Bestätigungscode, den du unten eingeben kannst."
```

Comment:
In most of the other translations I read the user is referenced as "Sie" for you, which is the formal form of the word, like you would talk to a stranger.

However, here you are using "du" for you, which is when you talk to a friend or person that is close to you.

Is there a common approach chosen here?

Reply:
I suspect DeepL returned more formal translations (for the new translated strings) because that is what it is trained to do. However, it seems that you're right that here, the translators who worked on it before chose the less formal phrasing.

I'm not sure what the best approach would be.

Reply:
For me personally, it would be fine to go with the less formal approach, and if that is what has been done before even better. But other opinions would be great as well.